uaclient/files.py

```diff
@@ -66,6 +66,9 @@ def read(self) -> Optional[str]:
     def delete(self):
         system.remove_file(self.path)
 
+    def is_present(self):
```

Comment:
This needs `@property` if we want to be able to access it like `file.is_present`

Reply:
Oh, my bad 🤦‍♂️🤦‍♂️